sentry-android-core/src/main/java/io/sentry/android/core/AnrIntegration.java

```diff
@@ -70,6 +71,10 @@ private void register(final @NotNull IHub hub, final @NotNull SentryAndroidOptio
           anrWatchDog.start();
 
           options.getLogger().log(SentryLevel.DEBUG, "AnrIntegration installed.");
+          final SdkVersion sdkVersion = options.getSdkVersion();
+          if (sdkVersion != null) {
+            sdkVersion.addIntegration("Anr");
+          }
```

Comment:
`m`: We added similar code for every integration. Maybe we can refactor this to get rid of the duplication. Ideally, we would somehow force adding an integration name when implementing an integration. Now you can easy forget to add this code snipped when adding a new integration. We could add an abstract class or an interface to enforce this.

Reply:
Yeah let's not introduce a breaking change now. We could do that for the next major perhaps.